Whats new july (#19801)
dotnet/docs: docs/whats-new/dotnet-docs-2020-07.md

> ---

Reviewer: Rename to just `2020-07.md`?

Based on @mairaw comment (https://github.com/dotnet/docs/pull/19257/#discussion_r447305779).

## .NET

### New articles

- [Choosing between anonymous and tuple types](/dotnet/standard/design-guidelines/choosing-between-anonymous-and-tuple) - Choosing between anonymous and tuple types
- [Cross-Platform Cryptography in .NET Core and .NET 5](/dotnet/standard/security/cross-platform-cryptography) - Cross-plat cryptography article
- [BinaryFormatter security guide](/dotnet/standard/serialization/binaryformatter-security-guide) - Ra off binary formatter

### Updated articles

- [Chaining tasks using continuation tasks](/dotnet/standard/parallel-programming/chaining-tasks-by-using-continuation-tasks) - Address concerns about chaining tasks and lacking continuation examples with .Unwrap()

## Community contributors

The following people contributed to the .NET docs during this period. Thank you! Learn how to contribute by following the links under "Get involved" in the [what's new landing page](index.yml).

- [Youssef1313](https://github.com/Youssef1313) - Youssef Victor (30)
- [NextTurn](https://github.com/NextTurn) - Next Turn (20)
- [pkulikov](https://github.com/pkulikov) - Petr Kulikov (10)
- [imba-tjd](https://github.com/imba-tjd) - 谭九鼎 (2)

> - [LauraKellerGitHub](https://github.com/LauraKellerGitHub) - Laura Keller (2)

Reviewer 2: This is a v-dash account.

```suggestion
```

---

> title: ".NET docs: What's new for June 28, 2020 - August 1, 2020"

Author: I think I should change this to the month:

```suggestion
title: ".NET docs: What's new for July 2020"
```

description: "What's new in the .NET docs for June 28, 2020 - August 1, 2020."
ms.date: 08/03/2020
---

> # .NET docs: What's new for June 28, 2020 - August 1, 2020

Author: same change:

```suggestion
# .NET docs: What's new for July 2020
```

> Welcome to what's new in the .NET docs from June 28, 2020 through August 1, 2020. This article lists some of the major changes to docs during this period.

Author: one more:

```suggestion
Welcome to what's new in the .NET docs from July 2020. This article lists some of the major changes to docs during this period.
```

- [Cross-Platform Cryptography in .NET Core and .NET 5](/dotnet/standard/security/cross-platform-cryptography) - Cross-plat cryptography article
- [BinaryFormatter security guide](/dotnet/standard/serialization/binaryformatter-security-guide) - Ra off binary formatter

### Updated articles

- [Chaining tasks using continuation tasks](/dotnet/standard/parallel-programming/chaining-tasks-by-using-continuation-tasks) - Address concerns about chaining tasks and lacking continuation examples with .Unwrap()

## Community contributors

The following people contributed to the .NET docs during this period. Thank you! Learn how to contribute by following the links under "Get involved" in the [what's new landing page](index.yml).

- [Youssef1313](https://github.com/Youssef1313) - Youssef Victor (30)
- [NextTurn](https://github.com/NextTurn) - Next Turn (20)
- [pkulikov](https://github.com/pkulikov) - Petr Kulikov (10)
- [imba-tjd](https://github.com/imba-tjd) - 谭九鼎 (2)
- [LauraKellerGitHub](https://github.com/LauraKellerGitHub) - Laura Keller (2)
- [tonytins](https://github.com/tonytins) - Anthony Leland (2)

> - [azure-sdk](https://github.com/azure-sdk) - Azure SDK Bot (1)

Reviewer 2: This is a Microsoft-owned service account.

```suggestion
```
